benchmark_runner/common/oc/oc.py

```diff
@@ -82,26 +75,29 @@ def get_worker_disk_ids(self):
         """
         workers_disk_ids = []
         if self.__worker_disk_ids:
-            workers_disk_ids = [disk_id for disk_list in self.__worker_disk_ids.values() for disk_id in disk_list]
+            for node, disk_ids in self.__worker_disk_ids.items():
+                for disk_id in disk_ids:
+                    workers_disk_ids.append(disk_id)
```

Comment:
Why are you changing this?  And you should raise an exception if there's nothing.

```
if self.__worker_disk_ids:
    return [disk_id for disk_list in self.__worker_disk_ids.values() for disk_id in disk_list]
else
    raise ...
```

Reply:
Again, make sure that you actually find a disk ID here.

Reply:
This really should raise an exception if nothing is found.  This is expected to find something; if it doesn't, it indicates that something is wrong here.